### PR TITLE
PY-60 implement client server connection

### PR DESCRIPTION
**Pull request by UlanDT**
- Update code structure
- Add settings
- Add client-server tcp connection

### CODE REVIEW

File: tests/cli/test_client.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""Module containing tests for client."""
+from tornado.testing import bind_unused_port
+
+from src.back.server import Server
+from src.cli.client import Client
+
+
+async def test_server() -> None:
```
Comment (WinterCitizen) [inline tests/cli/test_client.py:8]: Rename the test

**Review by WinterCitizen — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -43,10 +46,20 @@ line-length = 120
 exclude_lines = [
     "raise NotImplementedError",
     "if TYPE_CHECKING",
+    "if __name__",
+    "async def main",
     "pragma: no cover",
+    "import",
+    "Settings",
```
Comment (WinterCitizen) [inline pyproject.toml:53]: Why?

**Review by WinterCitizen — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (UlanDT) [reply pyproject.toml:53]: coverage is not ignoring imports and i don't think there should be tests for Settings instance

**Review by WinterCitizen — APPROVED** [review]

**Review by aalysher — APPROVED** [review]
lgtm

**Pull request merged**